### PR TITLE
Documentation site for tbds

### PR DESCRIPTION
**Pull request by wardpenney**
Super fancy documentation site! 

The outer frame has no styling, but `tbds` stylings are loaded for component examples.

In order to make an example, add a markdown file in the package. Include and reference any associated `html` examples from within the markdown file, using the Hugo `example` shortcode. 

screen shot 2018-03-30 at 5 48 33 pm

Future plans:

- render components in isolated iframes
- make the site frame look gud

### CODE REVIEW

File: documentation/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,101 @@
+env:
```
Comment (tysongach) [inline documentation/.eslintrc]: ✂️ ?

File: documentation/site/themes/tbds/layouts/_default/list.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{{ define "main" }}
+  <h2>
```
Comment (tysongach) [inline documentation/site/themes/tbds/layouts/_default/list.html]: Maybe swap the `h2` and `h1` in order? Would be odd to assistive technology to work backward in the tree.

File: packages/tbds-button/documentation/index.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+title: "tbds Button"
+---
+
+A great button for everyday use.
+
+{{< example fileName="example-1.html" >}}
+
```
Comment (tysongach) [inline packages/tbds-button/documentation/index.md]: ✂️ ?

File: documentation/.eslintrc (hunk 1 above)
Comment (wardpenney) [reply documentation/.eslintrc]: I think this has something to do with `eslint` that says it's in the browser, not on a node server?

File: documentation/site/themes/tbds/layouts/_default/list.html (hunk 2 above)
Comment (wardpenney) [reply documentation/site/themes/tbds/layouts/_default/list.html]: Actually, I'm going to remove that `h1` from the layout.

File: documentation/.eslintrc (hunk 1 above)
Comment (wardpenney) [reply documentation/.eslintrc]: Yeah it looks like it provides some global variables https://eslint.org/docs/user-guide/configuring#specifying-environments

**Pull request merged**